sparkmagic/sparkmagic/livyclientlib/livysession.py

```diff
@@ -59,6 +59,12 @@ def __init__(self, http_client, properties, ipython_display,
         super(LivySession, self).__init__()
         assert u"kind" in list(properties.keys())
         kind = properties[u"kind"]
+
+        if not should_heartbeat:
+            properties.pop(u"heartbeatTimeoutInSecond")
```

Comment:
pop will raise KeyError if not in the dictionary and a default is not given. Please add a default.